mne/epochs.py

```diff
@@ -691,7 +691,7 @@ def __init__(
             raw_sfreq = self.info["sfreq"]
         self._raw_sfreq = raw_sfreq
         self._check_consistency()
-        self.set_annotations(annotations)
```

Comment:
this is in `mne/epochs.py`... do we need the same fix for `Raw`?

Reply:
I don't think so, doing `Epochs(..., picks="subset")` is common- but I can't think of a similar case with `Raw`, beside I/O like in #12007 but I would argue this case should not happen, else it means the file you are trying to load is corrupted/erroneous (contains channel-specific annotations of non-existing channels).

Reply:
that makes sense, sorry, I wasn't realizing this was happening *in the constructor* (rather than in a method)